### PR TITLE
Refactoring code 

### PR DESCRIPTION
**Pull request by HSMathebula**
In this milestone I:

-Refactor code 'calculator.js' to the functional component.

- Use hooks

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,71 +1,74 @@
...
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { total: null, next: null, operation: null };
-  }
+const Calculator = () => {
+  // constructor(props) {
+  //   super(props);
+  //   this.state = { total: null, next: null, operation: null };
+  // }
```
Comment (ahmetbozaci) [inline src/components/calculator.js]: - [x] Please do not commit old pieces of code as inline comments. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @HSMathebula  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.




### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by HSMathebula** [comment]
Hi @ahmetbozaci thank you. fixed

**Review by bimbolabuari — APPROVED** [review]
Hi @HSMathebula, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**